### DISCUSSION TITLE
How does Ocelot use Consul cluster to discover services?

### DISCUSSION THREAD

**Discussion by jinyuttt** [ThreeMammals/Ocelot#1934] [Q&A]
## Expected Behavior / New Feature
Service discovery uses cluster configuration. How does Ocelot run? I see that Ocelot configuration is only an IP and port, so there is no cluster configuration using service discovery. For example, consul is clustered. How does Ocelot use consul cluster to discover services?

## Actual Behavior / Motivation for New Feature
Service discovery uses cluster configuration. How does Ocelot run? I see that Ocelot configuration is only an IP and port, so there is no cluster configuration using service discovery. For example, consul is clustered. How does Ocelot use consul cluster to discover services?

**Comment by raman-m** [comment]
@jinyuttt  Good question!

@ggnaegi We need your help!
Do you use Consul cluster setup in your Prod environment?
